### ISSUE TITLE
U3 - RT9193 - BP Pin left floating while datasheet especially states not to do that

### ISSUE THREAD

**Issue by p1ngb4ck** [makermoekoe/Reflow-Controller#8]
Hi !

As the title says. In your schematics, you leave the BP-Pin of IC U3 (RT9193_SC70-5) floating.
If you have a look at the datasheet of the RT9193 (https://www.richtek.com/assets/product_file/RT9193/DS9193-18.pdf), you will find the functional pin description on page 2 stating the following for the BP-Pin :
[quote]Reference Noise Bypass. This pin cannot be floating. Recommended to connect a 22nF or larger
ceramic capacitor between the BP and GND pins.[/quote]
Also, it is stated in the datasheet, how the actual pcb design around those capacitors should be handled for optimal performance (see page 8 of the mentioned datasheet) :
[quote]Bypass Capacitor and Low Noise
Connecting a 22nF between the BP pin and GND pin
significantly reduces noise on the regulator output, it is
critical that the capacitor connection between the BP pin
and GND pin be direct and PCB traces should be as short
as possible. There is a relationship between the bypass
capacitor value and the LDO regulator turn on time. DC
leakage on this pin can affect the LDO regulator output
noise and voltage regulation performance.[/quote]

But, rather than applying those changes, I propse a different approach :

As the RT9193 is hart to get in SC-70-5 packaging, I was actually looking for a direct replacement and found the SC573V33RTRC (available on mouser an others) that "should" (waiting for delivery of some other parts, so I can finally make & test the project) work as a direct replacement.
Advantage of that would be, that it is pin-compatible, is available .. and - doesn`t have a BP-Pin at all but just a 2nd gnd pin in its place - so with your current schematics, you would just leave that 2nd ground pin floating, which shouldn`t be a problem at all.

Thanks so much for all your work though, looking forward to build it =) !!!

**Comment by makermoekoe** [comment]
I think the words "cannot be floating" should mean that it "shouldn't be floating", because actually it works with leaving it floating.
Regarding the block diagram in the RT9193 datasheet, it is just to stabilize the reference voltage for the comparator and thus (as the datasheet says as well) is just to reduce the noise on the output voltage. Seems like the unstabilized output voltage is good enough for this use case, but of course the 22nF cap should be placed within the next revision.
Furthermore the SC573V33RTRC seems to be a good alternative for this revision.

**Comment by p1ngb4ck** [comment]
Yes you are absolutely right with shouldnt instead of cannot, I am sorry
Thanks so much for coming back on this topic and many many thanks for the in general awesome project.
Meanwhile I was able to successfully build it. So I can confirm the SC573V33RTRC is working in general - but it does not support enough power for wifi (at least during my testing).
In the meantime, I managed to get you code running on latest lib and framework versions (just very minor changes necessary) and I was able to add basic wifi and ota flashing.
I think this will be useful for ongoing developement, while having the board already installed to the oven. My goal is to  implement further wifi control (and temp-curve-modification/preset saving) via webserver on esp and sigrok.
Wifi and OTA already work, now going on to adding the actual wifi controls.
Thank you so much and please dont see this as criticism - the opposite is true indeed. I think this is such a nice project, that it is more than worth to invest my time into and I hope it is in your interest also, to see this being actively developed on =)

**Comment by choleaoum** [comment]
> SC573V33RTRC



> So I can confirm the SC573V33RTRC is working in general - but it does not support enough power for wifi (at least during my testing).

I can't find the R9193 available anywhere and the SC573 is not recommended for new design. Do you think RT9013-33GU5 would work and have enough power to support WiFI? 

I also wanted to say thanks for your fork.
